### PR TITLE
refactor styles

### PR DESCRIPTION
**Pull request by mkucaj1986**

### CODE REVIEW

File: src/components/AddLocation.vue [hunk 1]
```diff
@@ -85,14 +85,15 @@
 </script>
 
 <style rel="stylesheet/scss" lang="scss">
+	@import '../assets/styles/components/variables.scss';
```
Comment (mtrzesniowski) [inline src/components/AddLocation.vue]: Add one empty line below to separate imports from the proper styles

File: src/components/MyLocation.vue [hunk 2]
```diff
@@ -93,6 +93,7 @@
 </script>
 
 <style rel="stylesheet/scss" scoped lang="scss">
+	@import '../assets/styles/components/variables.scss';
```
Comment (mtrzesniowski) [inline src/components/MyLocation.vue]: Add one empty line below to separate imports from the proper styles

File: src/App.vue [hunk 3]
```diff
@@ -29,52 +29,20 @@
 
 <style lang="scss" rel="stylesheet/scss">
 	@import 'assets/styles/pgsStyles';
+	@import 'assets/styles/main.scss';
 
 	body {
-		font-family: 'WhitneySSm-Book', sans-serif;
+		font-family: $font-WhitneySSm-Book;
 		color: #333;
```
Comment (mtrzesniowski) [inline src/App.vue]: Make the color value a variable as well

File: src/components/Navbar.vue [hunk 4]
```diff
@@ -89,12 +89,14 @@
 </script>
 
 <style rel="stylesheet/scss" scoped lang="scss">
+	@import '../assets/styles/components/variables.scss';
+
 	.navbar-brand {
 		font-size: 22px;
```
Comment (mtrzesniowski) [inline src/components/Navbar.vue]: Please convert this to ems

File: src/components/Navbar.vue [hunk 5]
```diff
@@ -116,7 +118,7 @@
 			padding-top: 8px;
 
 			&-name {
-				font-family: 'WhitneySSm-Semibold', sans-serif;
+				font-family: $font-WhitneySSm-Semibold;
 				font-size: 16px;
```
Comment (mtrzesniowski) [inline src/components/Navbar.vue]: All of the font-size should be in ems

**Comment by mkucaj1986** [comment]
code review changes are ready  to check

**Pull request merged**